app/talk/inbox.cjsx

```diff
@@ -14,17 +16,21 @@ PAGE_SIZE = talkConfig.inboxPageSize
 
 promptToSignIn = -> alert (resolve) -> <SignInPrompt onChoose={resolve} />
 
+counterpart.registerTranslations 'en',
+  messagesPage:
+    title: 'Messages'
```

Comment:
This is inconsistent with the header and the url of the page. Maybe this should be `Inbox` instead?